src/components/HeaderSearch.js

```diff
@@ -28,20 +27,9 @@ export const HeaderSearch = ({ children }) => {
           icon={showSearchForm ? CloseIcon : SearchPlaceIcon}
           onClick={handleShowSearchForm}
         />
-        <Button
-          href="/settings"
-          className="btn btn__link--icon"
-          icon={SettingsIcon}
-          label="Settings button"
-        />
-        <Button
-          href="/help"
-          className="btn btn__link--icon"
-          icon={HelpIcon}
-          label="Help button"
-        />
-        {children}
+        {!showSearchForm ? <HeaderNav /> : null}
```

Comment:
Tiny thing, but this would be easier to read of you created a variable called:

`const showHeaderNav = !showSearchForm`

Then this could be:
```suggestion
        {showHeaderNav ? <HeaderNav /> : null}
```
You can also do `{ showHeaderNav && <HeaderNav /> }`

Reply:
Thanks, @laurenmbeatty! I created a `showHeaderNav` variable. I would like to use `&&` (AND operator). Because it's easier and simple. I agree with you. However, I remember that Katia shared with us an interesting [article](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) some time ago. Where author suggests that should use an "explicit" declaration.

But, if you would like that we will change it, we can do it 😄 .

Reply:
I agree good call!